great_expectations/data_context/store/expectations_store.py

```diff
@@ -111,6 +114,38 @@ def gx_cloud_response_json_to_object_dict(response_json: Dict) -> Dict:
 
         return suite_dict
 
+    def add_expectation(
+        self, suite: ExpectationSuite, expectation: Expectation
+    ) -> Expectation:
+        suite_identifier = self.get_key(suite=suite)
+        suite_dict = self.get(key=suite_identifier)
+        suite = ExpectationSuite(**suite_dict)
+
+        expectation.id = str(uuid.uuid4())
```

Comment:
Is it ok if we determine the ids that cloud uses for expectations?

Reply:
nope - we need to set the ID to `None` here so the backend knows its new [bdf6073](https://github.com/great-expectations/great_expectations/pull/9125/commits/bdf6073ba3da7de1e2836bf8480f3d8f24e6ffff)